### PR TITLE
Refs #31910 - don't assert duration of the agent ping test

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: test/models/ping_test.rb [hunk 1]
```diff
@@ -164,7 +164,6 @@ def test_ping_katello_agent_enabled
 
       assert_equal 'ok', result[:status]
       assert_equal '0 Processed, 0 Failed', result[:message]
-      assert_equal '0', result [:duration_ms]
```
Comment (ekohl) [inline test/models/ping_test.rb:167]: Does it make sense to assert it's present?
```ruby
assert_includes(result, :duration_ms)
```

Comment (evgeni) [reply test/models/ping_test.rb:167]: maybe? but we don't do that in the other tests either…

Comment (jturel) [reply test/models/ping_test.rb:167]: Perfectly find without it imo.

**Review by jturel — APPROVED** [review]
Thanks @evgeni !

**Pull request merged**